functions/profile.js

```diff
@@ -0,0 +1,107 @@
+const admin = require('./admin');
+const Promise = require('bluebird');
+const User = require("./user");
+
```

Comment:
In reading this code, I needed to write down the data structure for `updateProfileQueue` -- would be good to include in doc/updateProfileQueue.md and reference here).  Prolly needs better formatting than this, but I wanted to write this down to make sure I understood correctly.

```
/accounts/{userId}/
   githubToken: <set by client>
   profileNeedsUpdate: null for clients < 1.0.8, set to true by client or enqueueUpdateLegacy, set to false when profile is complete

/updateProfileQueue/{userId} -> function listens to this node, so triggers when either timestamp is written
   createdAt: timestamp when it was created by enqueueUpdate (when profileNeedsUpdate was first set to true)
   updatedAt: updated by cron to re-trigger the function
```

See note below, I think `updatedAt` should be written by `enqueueUpdate` as well, so we can exclude late-comers from the cron fix-up